planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -1095,6 +1095,9 @@ void AvoidanceModule::generateTotalShiftLine(
   // overwrite shift with current_ego_shift until ego pose.
   const auto current_shift = helper_.getEgoLinearShift();
   for (size_t i = 0; i <= avoidance_data_.ego_closest_path_index; ++i) {
+    if (sl.shift_line.size() <= i) {
+      break;
+    }
```

Comment:
```suggestion
  for (size_t i = 0; i < sl.shift_line.size(); ++i) {
    if (avoidance_data_.ego_closest_path_index < i) {
      break;
    }
    sl.shift_line.at(i) = current_shift;
    sl.shift_line_grad.at(i) = 0.0;
  }
```

Reply:
hmm, seems wrong

[174c903](https://github.com/autowarefoundation/autoware.universe/pull/4628/commits/174c90319fd10dbb75ea9824316f12ab2fc527a4)
[e142f52](https://github.com/autowarefoundation/autoware.universe/pull/4628/commits/e142f521ccf2f77fac1767b5eefa1da95a08ea0c)

Reply:
@satoshi-ota 
fixed in [2a855c9](https://github.com/autowarefoundation/autoware.universe/pull/4628/commits/2a855c9faf8e397e841bddad9d347a776112d44b)